Document new AWS regions / tidy up "Create a Cluster" (#559)
astronomer/docs: astro/install-aws.md

- Your preferred Astro Cluster name.
- The AWS region that you want to host your Cluster in.
- Your preferred node instance type.

> - Your preferred max node count.

Reviewer: ```suggestion
- Your preferred maximum node count.
```

> If not specified, we will create a Cluster with two `m5.xlarge` nodes in `us-east-1` by default. For information on all supported regions and configurations, see [AWS Resource Reference](resource-reference-aws.md).

Reviewer: ```suggestion
If not specified, we will create a Cluster with two `m5.xlarge` nodes in `us-east-1` and a maximum node count of 20 by default. For information on all supported regions and configurations, see [AWS Resource Reference](resource-reference-aws.md).
```

To provision additional Clusters, complete the setup in [Create a Cluster](create-cluster.md) after completing your initial installation.

:::caution

> Some AWS regions that Astronomer supports are disabled by default on AWS, including:
> 
> - `af-south-1` (Cape Town)
> - `ap-east-1` (Hong Kong)
> - `me-south-1` (Bahrain)
> 
> If you're setting up your first Cluster in any of these regions, you need to complete the additional setup as described in [Create a Cluster](create-cluster.md#edit-your-aws-trust-policy).
> 

Reviewer: ```suggestion
There are some AWS regions that require additional steps to enable on Astronomer. These regions are:

- `af-south-1` (Cape Town)
- `ap-east-1` (Hong Kong)
- `me-south-1` (Bahrain)
- `ap-northeast-3` (Osaka)

To create a Cluster in any of these regions, complete the additional steps described in [Create a Cluster](create-cluster.md#edit-your-aws-trust-policy).
```

To provision additional Clusters, complete the setup in [Create a Cluster](create-cluster.md) after completing your initial installation.

:::caution

Some AWS regions that Astronomer supports are disabled by default on AWS, including:

- `af-south-1` - Africa (Cape Town)
- `ap-east-1` - Asia Pacific (Hong Kong)
- `me-south-1` - Middle East (Bahrain)

> - `eu-south-1` - Europe (Milan)

Reviewer: ```suggestion
```

> If you're setting up your first Cluster in any of these regions, you need to complete the additional setup as described in [Create a Cluster](create-cluster.md#edit-your-aws-trust-policy).

Reviewer: ```suggestion
If you're setting up your first Cluster in any of these regions, you need to complete the additional setup as described in [Create a Cluster](create-cluster.md#additional-setup-for-aws-regions-that-are-disabled-by-default).
```

Edit path to doc here given new subsection?
